pkg/ocicni/ocicni.go

```diff
@@ -142,6 +142,9 @@ func (plugin *cniNetworkPlugin) monitorNetDir() {
 				logrus.Errorf("CNI setting failed, continue monitoring: %v", err)
 
 			case err := <-watcher.Errors:
```

Comment:
Does this really happen?

Reply:
I mean an event with nil value?

Reply:
@rajatchopra yes, it seems to, have reproduced on real clusters running RHEL too.